### PR TITLE
contact summary e2e tests angular 10

### PR DESCRIPTION
**Pull request by newtewt**
# Description

update contract summary tests to async/await and work with angular10 changes. 

https://github.com/medic/angular10-migration/issues/80

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

File: tests/e2e/contact-summary.js [hunk 1]
```diff
@@ -161,97 +163,64 @@ describe('Contact summary info', () => {
...
+    
+    expect(await contactsPo.cardFieldLabelText('test_sex')).toBe('test_sex');
+    expect(await contactsPo.cardFieldText('test_sex')).toBe(CAROL.sex);
 
-    expect(helper.getTextFromElement(element(by.css('.content-pane .meta > .card .col-sm-3:nth-child(3) label'))))
-      .toBe('aliceTag');
-    expect(helper.getTextFromElement(element(by.css('.content-pane .meta > .card .col-sm-3:nth-child(3) p'))))
-      .toBe(ALICE.name + ' ' + ALICE.phone);
+    expect(await contactsPo.cardFieldLabelText('alicetag')).toBe('aliceTag');
+    expect(await contactsPo.cardFieldText('alicetag')).toBe(ALICE.name + ' ' + ALICE.phone);
```
Comment (craig-landry) [inline tests/e2e/contact-summary.js:203]: I know it's preexisting stuff, but please update the syntax on this and the others below while you're running thought this bit of code.
```suggestion
    expect(await contactsPo.cardFieldText('alicetag')).toBe(`${ALICE.name} ${ALICE.phone}`);
```

File: tests/helper.js [hunk 2]
```diff
@@ -35,6 +43,20 @@ module.exports = {
       });
   },
 
+  clickElementNative: async element => {
+    handleUpdateModalNative();
+    try {
+      const msg = `Element taking too long to appear in the DOM ${element.locator()}`;
```
Comment (craig-landry) [inline tests/helper.js:49]: Thanks for using the template string here :)

File: tests/page-objects/contacts/contacts.po.js [hunk 3]
```diff
@@ -94,7 +104,7 @@ module.exports = {
   search: async query => {
     searchBox.clear();
     searchBox.sendKeys(query);
```
Comment (craig-landry) [inline tests/page-objects/contacts/contacts.po.js:106]: Both [clear](https://www.protractortest.org/#/api?view=webdriver.WebElement.prototype.clear) and [sendKeys](https://www.protractortest.org/#/api?view=webdriver.WebElement.prototype.sendKeys) look to return promises and you'll need to `await` them to be sure those keystrokes happen before the `click` on the following line. With that though, I would have expected this test to fail most or all of the time, so perhaps I'm missing something here. Is there something else involved that's causing these actions to happen in the desired sequence?
```suggestion
    await searchBox.clear();
    await searchBox.sendKeys(query);
```

File: tests/utils.js [hunk 4]
```diff
@@ -689,7 +689,6 @@ module.exports = {
         method: 'POST',
         body: { docs: docs }
       });
-
```
Comment (craig-landry) [inline tests/utils.js:692]: I kind of like this better with the newline separating the other lines. Just my personal preference though.

**Review by craig-landry — CHANGES_REQUESTED** [review]

File: tests/e2e/contact-summary.js [hunk 5]
```diff
@@ -1,5 +1,7 @@
 const utils = require('../utils');
 const helper = require('../helper');
+const contactsPo = require('../page-objects/contacts/contacts.po');
+
```
[the remark this answers is not in the document]
Comment (newtewt) [reply tests/e2e/contact-summary.js:4]: Are you referring to the blank line?

**Comment by newtewt** [comment]
@craig-landry  can you re-review. I believe I have addressed everything

File: tests/e2e/contact-summary.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (craig-landry) [reply tests/e2e/contact-summary.js:4]: Yeah, I thought there were 2 blank lines there.

File: tests/utils.js [hunk 6]
```diff
@@ -691,7 +691,7 @@ module.exports = {
...
+
```
Comment (craig-landry) [inline tests/utils.js:694]:
```suggestion

```

There are whitespaces here that should be removed. Chances are your editor slipped them in there to "help".

**Review by craig-landry — COMMENTED** [review]
Just a nitpick thing and otherwise all set here.

**Review by craig-landry — APPROVED** [review]

**Pull request merged**